apps/randomcats/random_cats.star

```diff
@@ -0,0 +1,54 @@
+"""
+Applet: Random Cats
+Summary: Shows pictures of cats
+Description: Shows random pictures of cats/gifs of cats.
+Author: mrrobot245
+"""
+
+load("encoding/json.star", "json")
+load("render.star", "render")
+load("schema.star", "schema")
+load("http.star", "http")
+
+def main(config):
+    if (config.bool("gifs") == False):
+        imgSrc = http.get("https://cataas.com/cat").body()
+    else:
+        imgSrc = http.get("https://cataas.com/cat/gif").body()
```

Comment:
It looks like you can add `width` and `height` query params to this url so that the returned image is already scaled down to the proper size. Doing that would reduce the payload size and speed things up.

i.e.
https://cataas.com/cat?height=32&?width=64
https://cataas.com/cat/gif?height=32&?width=64